internal/core/src/segcore/SegmentSealedImpl.cpp

```diff
@@ -1125,9 +1125,9 @@ SegmentSealedImpl::bulk_subscript_impl(int64_t element_sizeof,
 
 void
 SegmentSealedImpl::ClearData() {
-    field_data_ready_bitset_.clear();
-    index_ready_bitset_.clear();
-    binlog_index_bitset_.clear();
+    field_data_ready_bitset_.reset();
+    index_ready_bitset_.reset();
+    binlog_index_bitset_.reset();
```

Comment:
reset will reset the bitset into zero value for every bit.
clear will clear the underlying allocated space.
we need `reset` here.